omnigibson/utils/ui_utils.py

```diff
@@ -590,7 +590,7 @@ def __init__(self, robot):
                 "command_dim": controller.command_dim,
             }
             idx += controller.command_dim
-            for i in controller.dof_idx:
+            for i in controller.dof_idx.tolist():
```

Comment:
is this because it could be shape (1, N) instead of (N,)?

Reply:
It's just an error that happened in the torch conversion. `i` needs to be an int and not a tensor.